app/assets/stylesheets/helpers/_header.scss

```diff
@@ -88,10 +88,13 @@
       display: block;
       margin: 0;
       border: 0;
-      height: 35px;
+      min-height: 36px;
       padding: 6px 0;
-      @include ie-lte(7){
-        height: 23px;
+      @include ie-lte(8) {
```

Comment:
Good enough for me. Maybe do a PR for caniuse? Data file for `box-sizing` is https://github.com/Fyrd/caniuse/blob/master/features-json/css3-boxsizing.json